tests/index.test.js

```diff
@@ -57,7 +56,7 @@ testHelperFn('', '', 'decode empty string');
 testHelperFn(null, null, 'decode null');
 testHelperFn(undefined, undefined, 'decode undefined');
 testHelperFn(1234, 1234, 'decode number');
-testHelperFn('&nbsp;', ' ', 'decode &nbsp;');
+testHelperFn('&nbsp;', ' ', 'decode &nbsp;');
```

Comment:
i actually dont know what this was, had to copy a space from the output which to me looked the same but somehow it wasnt and it only worked when i copied it